addons/crm/views/crm_lead_views.xml

```diff
@@ -77,8 +76,9 @@
                                         <field class="mb-0" name="automated_probability" force_save="1"/> %
                                     </small>
                                     <div id="probability" class="d-flex align-items-baseline">
+                                        <field name="is_won" invisible="1"/>
```

Comment:
Could be put next to "is_lost" at the beginning of the sheet tag for better :socks: